feature-libs/product-configurator/common/components/configure-cart-entry/configure-cart-entry.component.ts

```diff
@@ -31,14 +31,18 @@ export class ConfigureCartEntryComponent {
   }
 
   /**
-   * Verifies whether the cart entry has an order code and returns a corresponding owner type.
+   * Determines owner for an entry (can be part of cart, order or quote)
    *
    * @returns - an owner type
    */
   getOwnerType(): CommonConfigurator.OwnerType {
-    return this.cartEntry.orderCode !== undefined
-      ? CommonConfigurator.OwnerType.ORDER_ENTRY
-      : CommonConfigurator.OwnerType.CART_ENTRY;
+    if (this.cartEntry.quoteCode !== undefined) {
+      return CommonConfigurator.OwnerType.QUOTE_ENTRY;
+    } else {
+      return this.cartEntry.orderCode !== undefined
+        ? CommonConfigurator.OwnerType.ORDER_ENTRY
+        : CommonConfigurator.OwnerType.CART_ENTRY;
+    }
```

Comment:
so if there is a quoteCode and orderCode, we still assume QUOTE_ENTRY. Is this desired and if yes a test for this case would be nice